site/content/en/docs/Getting started/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Getting started"
+linkTitle: "Getting started"
+weight: 4
+---
+
+AWS Copilot makes it easy to deploy your containers to AWS in just a few steps. In this tutorial we’re going to do just that - we’re going to deploy a sample front end service that you can visit in your browser. While we’ll be using a sample static website in this example, you can use AWS Copilot to build and deploy any container app with a Dockerfile. After we get your service all set up, we’ll show you how to delete the resources Copilot created to avoid charges.
+
+Sound fun? Let’s do it!
+
+#### Step 1: Download & Configure AWS Copilot
+
+You’ll need a few things to use AWS Copilot - the AWS Copilot binary, AWS CLI, Docker Desktop and AWS credentials. 
```

Comment:
Feel like a lot of potential customers will have problems that they don't have a well-configured AWS CLI and aws profile. Maybe we should put more details about it in our wiki/doc?

Reply:
Discussed offline, we'll make updates in a follow-up PR along with some CSS changes